<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tblVeThangNam">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>1001</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>IDMAVE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mã Phương Tiện</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Biển Số</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tên Chủ Xe</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Loại vé</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Giá</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ngày Bất Đầu</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ngày Hết Hạn</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Trạng Thái</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>60</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Loại vé</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Giá vé</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ngày bắt đầu </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>140</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ngày hết hạn</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtGiaVe">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbLoaiVe">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dtNgayBatDau">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dtNgayHetHan">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>130</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDangKy">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Đăng kí</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLamMoi">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Làm mới</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSua">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Sửa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnXoa">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Xóa</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMaVe">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>ID VE DANG KI</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbMaPhuongTien">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnThoat">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Thoát</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txttukhoa">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>230</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Tìm Kiếm </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btntimkiem">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>230</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Tìm Kiếm</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
